app-admin/keepassxc/keepassxc-2.7.10.ebuild

```diff
@@ -74,7 +74,7 @@ PATCHES=( "${FILESDIR}/${PN}-2.7.10-tests.patch" )
 
 src_prepare() {
 	if [[ "${PV}" != *_beta* ]] && [[ "${PV}" != *9999 ]] ; then
-		printf '%s' "${PV}" > .version || die
+		echo -n "${PV}" > .version || die
```

Comment:
dont see any reason to change this

Reply:
> It is shorter and IMO simpler. Should I revert?
> 
> Also if the newline is ignored the `-n` can be removed, further simplifying the line.

@fkobi 

test with simple `echo`, if build passes and `/usr/bin/keepassxc-cli --version` reports  `2.7.10` its fine